terraso_backend/apps/project_management/graphql/projects.py

```diff
@@ -216,14 +224,19 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
         user = info.context.user
         project_id = kwargs["id"]
         project = cls.get_or_throw(Project, "id", project_id)
-        if not user.has_perm(Project.get_perm("delete"), project):
+        if not check_project_permission(user, ProjectAction.DELETE, Context(project=project)):
             cls.not_allowed()
```

Comment:
Question: Should there be an argument passed in here? Or is that just part of #1264?